### PR TITLE
Closes #2270 - setops pytest benchmark

### LINKED ISSUE #2270: PyTest Benchmark for `setops`
Configure pytest benchmark for `segarr_setops.py`, `setops.py`, and `setops_multiarray.py`

### PR DESCRIPTION
**Pull request by joshmarshall1**
This PR Closes #2270 

Pytest Benchmark updates for the `segarr_setops.py`, `setops.py`, and `setops_multiarray.py` benchmark

### CODE REVIEW

File: benchmark_v2/setops_benchmark.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+@pytest.mark.benchmark(group="Numpy_Setops")
+@pytest.mark.parametrize("op", OPS1D)
+@pytest.mark.parametrize("dtype", TYPES)
+def bench_np_setops(benchmark, op, dtype):
+    """
+    Measures performance of Numpy setops for comparison
+    """
+    if pytest.numpy:
+        seed = pytest.seed
```
Comment (pierce314159) [inline benchmark_v2/setops_benchmark.py:89]: also im less sure of this one but would the `if dtype in pytest.dtype:` useful here too?

File: benchmark_v2/setops_benchmark.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+@pytest.mark.benchmark(group="AK_Setops")
+@pytest.mark.parametrize("op", OPS1D)
+@pytest.mark.parametrize("dtype", TYPES)
+def bench_ak_setops(benchmark, op, dtype):
+    """
+    Measures the performance of arkouda setops
+    """
+    cfg = ak.get_config()
+    N = pytest.prob_size * cfg["numLocales"]
```
[the remark this answers is not in the document]
Comment (Ethan-DeBandi99) [reply benchmark_v2/setops_benchmark.py:60]: Tagging on here, yes the `dtype` check is needed here

File: benchmark_v2/setops_benchmark.py (hunk 1 above)
Comment (Ethan-DeBandi99) [reply benchmark_v2/setops_benchmark.py:89]: Again tagging on, yes the `dtype` check is needed in addition to the `numpy` check.

**Review by Ethan-DeBandi99 — CHANGES_REQUESTED** [review]

**Review by Ethan-DeBandi99 — APPROVED** [review]

File: benchmark_v2/setops_benchmark.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ethan-DeBandi99) [reply benchmark_v2/setops_benchmark.py]: I pushed the update

**Review by jaketrookman — APPROVED** [review]

**Review by pierce314159 — APPROVED** [review]

**Pull request merged**